modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/CruiseSign.java

```diff
@@ -58,7 +59,8 @@ public final void onSignClick(PlayerInteractEvent event) {
             sign.setLine(0, "Cruise: ON");
             sign.update(true);
 
-            c.setCruiseDirection(sign.getRawData());
+            org.bukkit.material.Sign materialSign = (org.bukkit.material.Sign) event.getClickedBlock().getState().getData();
+            c.setCruiseDirection(CruiseDirection.fromBlockFace(materialSign.getFacing()));
```

Comment:
How does this deal with standing vs wall signs?  We'd like standing signs to always have a null (0x0) cruise direction and wall signs to have their respective direction.

Reply:
Currently crafts cruise in the direction of standing signs unless they are diagonal in which case they will not move horizontally